packages/core/src/routes/session.ts

```diff
@@ -112,6 +113,48 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     return next();
   });
 
+  router.post(
+    '/session/register',
```

Comment:
maybe `POST /session/user`?

Reply:
I think maybe `POST /session/user` is left to "create users by admin".
Registering contains the redirect logic and Creating users does not.

Reply:
admin don't use `session`